ui/imports/shared/popups/send/views/AmountToSend.qml

```diff
@@ -145,9 +149,17 @@ ColumnLayout {
                     errorMessage: ""
 
                     validate: (text) => {
-                                  var num = LocaleUtils.numberFromLocaleString(topAmountToSendInput.text,
+                                  const num = LocaleUtils.numberFromLocaleString(topAmountToSendInput.text,
                                                                                topAmountToSendInput.locale)
-                                  return !isNaN(num) && num > 0 && num <= root.maxInputBalance
+                                  if (isNaN(num) || num <= 0 || num > root.maxInputBalance) {
+                                      return false
```

Comment:
Looks good... but I'm not sure we need the same (strict) rules in the context of SwapModal CC @Khushboo-dev-cpp

Reply:
yes that true, I think it should be the same! Anyways it would be great to get QA testing for this PR for all Modals that use this component.
which is Send and Swap Modal.

Reply:
Also I know you did not implement that part but `LocaleUtils.numberFromLocaleString(topAmountToSendInput.text,  topAmountToSendInput.locale) `can lead to lossy conversion from string to float 
we should instead use `AmountsArithmetic.fromString(stringNumber)` would be ideal

https://www.notion.so/Strategy-for-Numbers-and-Strings-in-Swap-and-Generally-5fed06457c134c82ab893e2b0da03c6e